components/infobox/commons/infobox_league.lua

```diff
@@ -19,6 +19,7 @@ local Variables = require('Module:Variables')
 local Locale = require('Module:Locale')
 local Page = require('Module:Page')
 local LeagueIcon = require('Module:LeagueIcon')
+local WarningBox = require('Module:WarningBox')
```

Comment:
Shouldn't changes like this to the base module be requested via a separate PR?

Reply:
Added via #884